File: Phase-1-Foundations/Java/Day027_RightTriangleStarPattern.java
package Phase_1_foundations;

import java.util.Scanner;

/**
 * Headline: Right Triangle Star Pattern
 * Description: Print a right-angled triangle of stars with N rows.
 *
 * Pseudocode:
umput(num)

for row = 1; i <= num; i++:
  for column = 1; column <= row; column++:
    print("*")
  print()
 */

// ---- Program ----
public class Day027_RightTriangleStarPattern {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of rows: ");
        int num = sc.nextInt();

        for (int row = 1; row <= num; row++) {

            for (int column = 1; column <= row; column++) {
                System.out.print("* ");
            }

            System.out.println();
        }

        sc.close();
    }
}

File: Phase-1-Foundations/Python/Day027_RightTriangleStarPattern.py
"""
Headline: Right Triangle Star Pattern
Description: Print a right-angled triangle of stars with N rows.

Pseudocode:
inupt(num)
for row in range(1, num + 1):
  for column in range(1, row + 1):
    print("*")
  print()
"""

# ---- Program ----
num = int(input("Enter number of rows: "))

for row in range(1, num + 1):
    for cloumn in range(1, row + 1):
        print("*", end=" ")
    print()